ghost/admin/app/controllers/explore.js

```diff
@@ -1,13 +1,23 @@
 import Controller from '@ember/controller';
 import {action} from '@ember/object';
+import {isEmpty} from '@ember/utils';
 import {inject as service} from '@ember/service';
+import {task} from 'ember-concurrency';
 
 export default class ExploreController extends Controller {
     @service explore;
     @service router;
+    @service store;
+
+    integrations = null;
+
+    constructor() {
+        super(...arguments);
+        this.initIntegrations.perform();
```

Comment:
For controllers it's usually best to avoid data loading in a constructor because we're not in control of when a controller is initialised. If you want models to be loaded it would be better to keep that in the route